CHANGELOG.md

```diff
@@ -2,6 +2,58 @@
 
 - Fix path to Sinon in browser environments
 
+# [5.0.0](https://github.com/goodeggs/goodeggs-test-helpers/compare/v4.0.0...v5.0.0)
+
+### BREAKING CHANGES
+- Sinon no longer included. `yarn add sinon@latest` and change `import {sinon} from 'goodeggs-test-helpers'` to `import sinon from 'sinon'`
+- `useSinonSandbox()` removed. Use sinon directly:
+For stubbing, spying, and stubbing logger:
+```
+import sinon from 'sinon';
```

Comment:
Because sandboxes are just sinon instances, I generally prefer importing the sandbox itself and naming the sandbox instance `sinon`, e.g.:

```js
import {createSandbox} from 'sinon';

// ...

const sinon = createSandbox();
afterEach('restore sandbox', () => sandbox.restore());
```

It prevents me from accidentally using the un-sandboxed version of Sinon and I usually don't think "I need a stub from the sandbox;" instead, I generally think "I need stubs, Sinon gives me stubs".

Reply:
Also, I think you're supposed to use `sinon.sandbox.create` in sinon 5.x? Or do I have that reversed? Anyway, maybe worth a callout if you're supposed to do it differently between 4.x and 5.x.

Reply:
I called it sandbox to match the sinon docs. The advantage of the word `sandbox` over `sinon` is that `sandbox` implies that it will be automatically restored afterEach. PS the interface changed quite a bit after talking to Phil.

Reply:
Sandbox doesn't necessarily imply this, though—it doesn't get restored afterEach unless you remember to do it, and you don't necessarily have to use it that way!